ocdskingfisherprocess/settings/base.py

```diff
@@ -147,6 +147,9 @@
     "project": "kingfisher",
 }
 
+# Kingfisher Collect scraped files
+FILES_STORE = os.getenv("FILES_STORE", '/data')
```

Comment:
```suggestion
# The directory from which to read the files written by Kingfisher Collect. If Kingfisher Collect and Kingfisher
# Process share a filesystem, this will be the same value for both services.
FILES_STORE = os.getenv("FILES_STORE", "/data")
```